grammars/hrm.cson

```diff
@@ -32,29 +32,25 @@
         'name': 'keyword.control.c'
   }
   {
-    'match': '^    (COMMENT) +([0-9]+)'
+    'match': '^ *(COMMENT) +([0-9]+)'
     'captures':
       '1':
         'name': 'entity.name.function.preprocessor.c'
       '2':
-        'name': 'string.quoted.double.include.c'
+        'name': 'constant.numeric.c'
```

Comment:
I don't understand the reason for this change.  Can you explain it a little more for me?  Is it to match up with the changes below?